libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts

```diff
@@ -0,0 +1,32 @@
+import {
+    Component,
+    Input,
+    ChangeDetectionStrategy,
+    OnChanges
+  } from '@angular/core';
+  import { FormGroup } from '@angular/forms';
+  import { FormlyFieldConfig } from '@ngx-formly/core';
+ 
+  @Component({
+    selector: 'sds-filters',
+    template: `
+    <sds-accordion multi="true">
+        <sds-accordion-item>
+            <sds-accordion-item-header> {{accordionLable}} </sds-accordion-item-header>
+            <formly-form [form]="form" [fields]="fields" [model]="model"></formly-form>
+        </sds-accordion-item>
+    </sds-accordion>
+    `,
+    changeDetection: ChangeDetectionStrategy.OnPush
+  })
+  export class SdsFiltersComponent implements OnChanges{
+    @Input() public form: FormGroup;
+    @Input() public fields: FormlyFieldConfig[];
+    @Input() public model: any;
```

Comment:
Please Add comments on the properties and remove unused method

Reply:
Please Add comments on the properties and remove unused method? And for
 @Input() public model: any; How do you know the model has changed?

Reply:
@thomastighe Updated comments

Reply:
The formly automatically updates the model, we need to subscribe the model to track the changes. I am updating the example

Reply:
So the model is of type observable<any> not just any?

Reply:
It doesn't need to be observable..

Reply:
So it can be of any type but it can be subscribed to?

Reply:
You don't have to subscribe, you can directly access model

Reply:
added example in side menu filters how model changes..

Reply:
If this works than I have no issues besides what Dave brought up. I thought it would need to be event driven.

Reply:
This is working fine, i have tested in prototype.

Reply:
Please add comments to the public items.

Reply:
Already updated comments long back.